libs/nx/src/nx.ts

```diff
@@ -28,51 +28,75 @@ interface WorkspaceProject {
 }
 
 export async function getNxProjects(cwd: string): Promise<WorkspaceProject[]> {
+  const nxProjects = await getNxProjectJsonProjects(cwd);
+  const workspaceProjects = await getNxWorkspaceProjects(cwd);
+
+  const additionalWorkspaceProjects = workspaceProjects.filter(
+    (proj) =>
+      nxProjects.find((nested) => nested.name === proj.name) === undefined
+  );
+
+  nxProjects.push(...additionalWorkspaceProjects);
+  return nxProjects;
+}
```

Comment:
```suggestion
  const nxProjects = await getNxProjectJsonProjects(cwd);
  const workspaceProjects = await getNxWorkspaceProjects(cwd);

  const relevantWorkspaceProjects = workspaceProjects.filter(
    (proj) =>
      nxProjects.find((nested) => nested.name === proj.name) == null
  );
  
  return [
     ...nxProjects,
     ...relevantWorkspaceProjects
  ];
}
```

Reply:
Naming things is hard 😉

I didn't want to create another array at the end of the function to minimise the allocations, but I suppose it's not that critical in CLI.

I'll update the code when I have access to the PC.

Thanks for review 😉